widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/defaulttheme/Chat.kt

```diff
@@ -86,10 +86,10 @@ private fun ChatOperatorMessageTheme(
 private fun ChatVisitorMessageTheme(
     pallet: ColorPallet
 ): MessageBalloonTheme? = pallet.run {
-    composeIfAtLeastOneNotNull(primaryColorTheme, baseDarkColorTheme, baseNormalColorTheme) {
+    composeIfAtLeastOneNotNull(primaryColorTheme, baseLightColorTheme, baseNormalColorTheme) {
         MessageBalloonTheme(
             background = LayerTheme(fill = primaryColorTheme),
-            text = TextTheme(textColor = baseDarkColorTheme),
+            text = TextTheme(textColor = baseLightColorTheme),
```

Comment:
Can't preview images in the related ticket but I assume it is okay that those colors switched places?

Reply:
Yes! That was one of the reasons for this fix.